src/api/app/views/webui2/webui/search/_results_owner.html.haml

```diff
@@ -8,7 +8,7 @@
         in #{result.rootproject}
 
       %ul.list-unstyled
-        - users_and_groups = result.users.merge(result.groups) { |_role, user, group| user.concat(group) }
+        - users_and_groups = (result.users || {}).merge(result.groups || {}) { |_role, user, group| user.concat(group) }
```

Comment:
I cannot say that I'm the biggest fan of this solution, but the other possibility I see is having to patch this in multiple methods of [src/api/app/models/owner_search/assignee.rb](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/owner_search/assignee.rb) wherever the owner is set.

Reply:
but having a function in owner.rb sounds still easier to parse than this - IMO